README.md

```diff
@@ -126,3 +126,72 @@ Now, for any new html page, we need to do the following:
 {% endblock %}
 
 ```
+
+## pre deployment to fly.io
+
+
+- Setting Up environmental variables
+
+  - Add `python-dotenv>=0.21.0` to requirements.txt
```

Comment:
Hold one, didn't you do that already?

Reply:
i did, i just updated the read me  file here

Reply:
But the README file should be instructions for a developer. Not stuff that is already done. Please make the README as easy and idiot-proof as possible. It should be possible to copy and paste the minimum number of commands and get everything working for any dev